### ISSUE TITLE
Operation basic.publish caused a connection exception unexpected_frame

### ISSUE THREAD

**Issue by mastak** [Polyconseil/aioamqp#76]
I have a big stream of data, and firstly I tried to push one data item in one `publish` call.
But periodically happened this error:

```
Server closed connection: UNEXPECTED_FRAME - expected content body, got non content body frame instead, code=505, class_id=60, method_id=40
```

In server logs:

```
Error on AMQP connection <0.11901.881> (172.30.0.83:56306 -> 172.17.0.26:5672, vhost: 'sync', user: 'producer', state: running), channel 1:
operation basic.publish caused a connection exception unexpected_frame: "expected content body, got non content body frame instead"
```

It was about 1000 - 10 000 items / sec.

Then I started making groups and send it (100 - 300 items / sec). An error has gone.
Is it has limit? :)
How can I fix it?

**Comment by dzen** [comment]
Hi @mastak,

Is this issue related to the encoding error ? c.f. https://github.com/Polyconseil/aioamqp/pull/78 ?

**Comment by mastak** [comment]
Hi @dzen,
I don't know. I fixed it by grouping data items in batch.)

I would like to test without batch next week,if you don't mind.

**Comment by dzen** [comment]
Sure, we all have a lots of work somewhere else ;)

**Comment by dzen** [comment]
Fixed with 09ec24d88a3fdf5848050a15352efd45cb92ac67

**Comment by RemiCardona** [comment]
Thanks @dnp1 for the heads up. Closing.
